scribejava-apis/src/main/java/com/github/scribejava/apis/FacebookApi.java

```diff
@@ -42,4 +52,32 @@ public String getAuthorizationUrl(final OAuthConfig config) {
         }
         return sb.toString();
     }
+
+    @Override
+    public OAuth20Service createService(OAuthConfig config) {
+        return new FacebookOAuth20ServiceImpl(this, config);
+    }
+    
+    private static class FacebookOAuth20ServiceImpl extends OAuth20Service {
```

Comment:
I made it a private class because the overridden method returns an abstract class and the concrete implementation is not used or needed anywhere else in the code. So it was just an attempt to keep the code organized and encapsulated. I can easily change that if you prefer.